deployments/consul-envoy/README.md

```diff
@@ -0,0 +1,79 @@
+# Monitor Consul Connect with Envoy Proxy
+
+Consul Connect emits metrics in statsd format to help understand the network. To report the metrics to SignalFx, you need to configure Consul Connect proxy to send the metrics to StatsD sink exposed by the agent.
+
+First, Configure Consul Connect Proxy to emit StatsD metrics.
+See example Consul service definition below.
+
+```hcl
+services {
+  name = "my_service"
+  port = 8080
+  connect {
+    sidecar_service {
+      proxy {
+        config {
+          envoy_extra_stats_sinks_json = <<EOF
+{
+  "name": "envoy.statsd", 
+  "config": {
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 8125,
+        "protocol": "UDP"
+      }
+    },
+    "prefix": "consul.my_service_mesh.my_service"
+  }
+}
+EOF
+        }
+      }
+    }
+  }
+}
+services {
+  name = "my_service2"
+  port = 9080
+  connect {
+    sidecar_service {
+      proxy {
+        config {
+          envoy_extra_stats_sinks_json = <<EOF2
+{
+  "name": "envoy.statsd", 
+  "config": {
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 8125,
+        "protocol": "UDP"
+      }
+    },
+    "prefix": "consul.my_service_mesh.my_service2"
+  }
+}
+EOF2
+        }
+      }
+    }
+  }
+}
+```
+
+It is mandatory to provide mesh/service name through prefix as shown since Consul does not specify the source of metric as part of the metric name. The source information will be parsed into dimensions by setting patterns in the agent configuration. For example:
+
+```yaml
+monitors:
+  - type: statsd
+    listenPort: 8125
+    # metrics need to be tagged with `plugin: consul`
+    extraDimensions:
+      plugin: consul
+    converters:
+    - pattern: "consul.{mesh}.{service}.cluster.{}.{action}"
+      metricName: "{action}"
+```
+
+For more information about StatsD monitor configuration, see the documentation on [StatsD monitor](../../docs/monitors/statsd.md).
```

Comment:
Don't make relative links like this going two levels back, it wreaks havoc on links when they get synced to the product docs site.  Just link it straight to the content on the master branch on github in the agent repo.